thefuntasty_danger.gemspec

```
@@ -12,8 +12,8 @@ Gem::Specification.new do |spec|
 
   spec.files       = ['Dangerfile']
 
-  spec.add_runtime_dependency 'danger', '~> 5'
-  spec.add_runtime_dependency 'danger-commit_lint', '~> 0'
-  spec.add_runtime_dependency 'danger-xcode_summary', '~> 0'
-  spec.add_runtime_dependency 'danger-swiftlint', '~> 0'
+  spec.add_runtime_dependency 'danger'
+  spec.add_runtime_dependency 'danger-commit_lint'
+  spec.add_runtime_dependency 'danger-xcode_summary'
+  spec.add_runtime_dependency 'danger-swiftlint'
```

If we remove these requirement then future versions of dependencies can break the gem, Gemfile.lock is not taken into account when installing gems.

And if you increase the spec version to 0.4.0, I'll release it to RubyGems.

https://rubygems.org

```suggestion
  spec.add_runtime_dependency 'danger', '~> 6'
  spec.add_runtime_dependency 'danger-commit_lint', '~> 0'
  spec.add_runtime_dependency 'danger-xcode_summary', '~> 0'
  spec.add_runtime_dependency 'danger-swiftlint', '~> 0'
```